lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -140,6 +145,70 @@ def test_unicode_user_partitions(self):
         _groups = [group.name for group in self.course.user_partitions[0].groups]
         self.assertEqual(_groups, user_groups)
 
+    def test_content_group_partition(self):
+        """
+        Test that cohort-schemed user partitions are ignored in the
+        grades export.
+        """
+        # Set up a course with 'cohort' and 'random' user partitions.
+        content_group_partition = UserPartition(
+            0,
+            'Content Group Configuration',
+            'Group Configuration for Content Groups',
+            [Group(0, 'Content Group A'), Group(1, 'Content Group B')],
+            scheme_id='cohort'
+        )
+        experiment_group_a = Group(2, 'Experiment Group A')
+        experiment_group_b = Group(3, 'Experiment Group B')
+        experiment_partition = UserPartition(
+            1,
+            'Content Experiment Configuration',
+            'Group Configuration for Content Experiments',
+            [experiment_group_a, experiment_group_b],
+            scheme_id='random'
+        )
+        course = CourseFactory.create(user_partitions=[content_group_partition, experiment_partition])
+
+        # Create user_a and user_b which are enrolled in the course
+        # and assigned to experiment_group_a and experiment_group_b,
+        # respectively.
+        user_a = UserFactory.create(username='user_a')
+        user_b = UserFactory.create(username='user_b')
+        CourseEnrollment.enroll(user_a, course.id)
+        CourseEnrollment.enroll(user_b, course.id)
+        course_tag_api.set_course_tag(
+            user_a,
+            course.id,
+            RandomUserPartitionScheme.key_for_partition(experiment_partition),
+            experiment_group_a.id
+        )
+        course_tag_api.set_course_tag(
+            user_b,
+            course.id,
+            RandomUserPartitionScheme.key_for_partition(experiment_partition),
+            experiment_group_b.id
+        )
+
+        # Verify that we see user_a and user_b in their respective
+        # content experiment groups, and that we do not see any
+        # content groups.
```

Comment:
I don't see that you are checking "we do not see any content groups". 

I also think it would be good to add at least one of the users to one of the groups in the cohort scheme (content group) user partition. Though perhaps you only care whether or not a column header is created for the cohort scheme partition, and whether or not a user is in one of the groups may not be relevant.